logstash-core-event/spec/logstash/event_old_api_spec.rb

```diff
@@ -0,0 +1,662 @@
+# encoding: utf-8
+require "spec_helper"
+require "logstash/util/decorators"
+require "json"
+
+describe LogStash::Event do
+
+  shared_examples "all event tests" do
+    context "[]=" do
+      it "should raise an exception if you attempt to set @timestamp to a value type other than a Time object" do
+        expect{subject["@timestamp"] = "crash!"}.to raise_error(TypeError)
+      end
+
+      it "should assign simple fields" do
+        expect(subject["foo"]).to be_nil
```

Comment:
why can't we test for `subject.get()` and `subject.set(x, y)`. There is already tons of tests for `subject[a]` and `[]=` correct?